### ISSUE TITLE
Enable @localType (and @localTypeDeclarationReference) in <reference>

### ISSUE THREAD

**Issue by LI-Yanlin** [SAA-SDT/eas-schemas#116]
## Creator of issue

1. Finna Services from The National Library of Finland (finna.fi)
2. piia.naukkarinen@helsinki.fi
3.
4.

## The issue relates to

- [ ] EAC-CPF schema issue
- [ ] EAC-CPF Tag Library issue
- [ ] EAD schema issue
- [ ] EAD Tag Library issue
- [x] Schema issue
- [ ] Tag Library issue
- [ ] Suggestions for all schemas
- [ ] Suggestions for all Tag Libraries
- [ ] Other

## Suggested Solution

* Text:  In our service the possibility to connect research publications and other materials to the archival material having used is important. Direct links (both PIDs and https’) should be able to bring in EAD4 to market the archival material for users. That is: `<bibref>` (or bibliographic reference of some kind) and `<archref>` are important.

Also, since students and researchers use our service to find archival materials (for research), form of citation information and its permanence is essential.

**Comment by MicheleCombs** [comment]
I agree that bibref and arcref are important.  we use arcref in particular extensively.

**Comment by kerstarno** [comment]
Copied from #140 for reference:
In `<reference>`, `@localType` would allow the definition of a typology that replaces and deepens the distinction between 'archival resource' / 'published resource' lost with the disappearance of `<archref>` and `<bibref>`.

**Comment by MicheleCombs** [comment]
Or, y'know, we could just keep archref and bibref which would serve the purpose and require less work when migrating to EAD 4...

**Comment by kerstarno** [comment]
TS-EAS discussed this during their meeting on 12/13 August and have come to the following conclusions:
* We will keep the general element name in line with the approach taken in other contexts and in light of reducing the amount of elements generally
* We will enable `@localType` with `<reference>`  
* We will not have a `@referenceType` attribute with a suggested “EASList” as it will not be possible to predict usefully a list of values that would apply in this case respectively such a list is likely to be seeing lots of regular requests for being extended
* We will create appropriate encoding examples to address the cases brought forward in SAA-SDT/EAS-TagLibraries#158 (`<p>` with several `<reference>`-s and using `@id` and `@target` to signify if `<reference>`-s relate to each other in some way) and SAA-SDT/EAS-TagLibraries#159 (ideal use case for `<relations>` in `<publicationNote>` rather than `<reference>`)

Changes required for the EAD 4.0 schema:
* Add `@localType` and `@localTypeDeclarationReference` to `<reference>`

**Comment by martincritelli** [comment]
Tested with XSD and RNG and everything works as expected.
